perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -266,22 +266,22 @@ def CreateScratchDisk(self, disk_spec):
     Args:
       disk_spec: virtual_machine.BaseDiskSpec object of the disk.
     """
-    volume = aws_disk.AwsDisk(disk_spec, self.zone)
-    self.scratch_disks.append(volume)
-
-    if volume.disk_type == disk.LOCAL:
-      if self.local_drive_counter >= self.max_local_drives:
+    # Instantiate the disk(s) that we want to create.
+    if disk_spec.disk_type == disk.LOCAL:
```

Comment:
This allows multiple non local disks to be striped together - you just specify --scratch_disk_type=remote_ssd and --num_striped_drives=2 in order to stripe together 2 ebs gp disks